10.0/contrib/wfbin/standalone.conf

```diff
@@ -91,9 +86,25 @@ if [ -z "$OPENSHIFT_SMTP_HOST" ]; then
     export OPENSHIFT_SMTP_HOST="localhost"
 fi
 
-max_memory_mb=${OPENSHIFT_GEAR_MEMORY_MB} 
-if [ -z "$OPENSHIFT_GEAR_MEMORY_MB" ]; then
-  max_memory_mb=512
+if [ -z $JAVA_GC_OPTS ]; then
+  JAVA_GC_OPTS="-XX:+UseParallelGC -XX:MaxPermSize=100m -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90"
```

Comment:
Can the user vary jdk7 vs. jdk8 with this image?  If only jdk8, MaxPermSize is ignored, and a benign by noisy info log shows up.